### ISSUE TITLE
Just one question

### ISSUE THREAD

**Issue by Eldenroot** [Sama34/OUGC-Signature-Control#3]
Hi @Sama34 - any specific reason why not to add your awesome plugins from GitHub to the MyBB mod site? Many users dont know about them, which is not good. They are really useful! Thank you!

**Comment by Sama34** [comment]
Yes, some of my plugins are not available in the official mods site, but I always try to keep them in sync.
